openslides/motions/urls.py

```diff
@@ -14,4 +14,5 @@
     url(r'^poll/(?P<poll_pk>\d+)/print/$',
         views.MotionPollPDF.as_view(),
         name='motionpoll_pdf'),
+    url(r'^encode_media/', views.encode_media, name="media_encoding")
```

Comment:
Please add blank line befor this and a comma at the end of the line.

Mayba move this view to the mediafiles app.